Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -487,12 +508,21 @@
   "block.biomeswevegone.overgrown_stone": "Заросший камень",
   "block.biomeswevegone.packed_black_ice": "Упакованный чёрный лёд",
```

Не "упакованный", а "плотный"
```suggestion
  "block.biomeswevegone.packed_black_ice": "Упакованный плотный лёд",
```